lms/static/js/ccx/view/schedule_tree.js

```diff
@@ -0,0 +1,195 @@
+            chapters: this.chapters,
+            templateTreeNode: _.template(scheduleTreeNodeTemplate)
+          }));
+          this.configureEditDateViews();
+        }
+        return this;
+      },
+
+      configureEditDateViews: function() {
+        var self = this;
+        this.$('table.ccx-schedule .due-date').each(function() {
+          var date = $(this).data('date');
+          var scheduleDateButtonView = new DateButtonView({
+            el: $(this),
+            date: date,
+            dateType: 'due'
+          });
+          scheduleDateButtonView.render();
+          self.listenTo(
+            scheduleDateButtonView,
+            'updateDate',
+            self.updateDate
+          );
+        });
+
+        this.$el.find('table.ccx-schedule .start-date').each(function() {
+          var date = $(this).data('date');
+          var scheduleDateButtonView = new DateButtonView({
+            el: $(this),
+            date: date,
+            dateType: 'start'
+          });
+          scheduleDateButtonView.render();
+          self.listenTo(
+            scheduleDateButtonView,
+            'updateDate',
+            self.updateDate
+          );
+        });
+      },
+
+      updateDate: function(dateType, newDate, location) {
+        var path = location.split(' ');
+        var chapterLocation = path[0];
+        var sequentialLocation = path[1];
+        var verticalLocation = path[2];
+
+        this.trigger(
+          'applyUnitToScheduleTree',
+          dateType,
+          newDate,
+          chapterLocation,
+          sequentialLocation,
+          verticalLocation
+        );
+      },
+
+      removeUnitFromTree: function(e) {
+        // remove a unit which can be chapter, sequential
+        // or vertical from schedule tree
+        e.preventDefault();
+        var target = e.target || e.srcElement;
+        var location = $(target).data('location');
+        var path = location.split(' ');
+        var chapterLocation = path[0];
+        var sequentialLocation = path[1];
+        var verticalLocation = path[2];
+
+        this.trigger(
+          'hideUnitFromScheduleTree',
+          chapterLocation,
+          sequentialLocation,
+          verticalLocation
+        );
+      },
+
+      removeAllUnitFromTree: function() {
+        this.trigger('hideAllUnitFromScheduleTree');
+      },
+
+      toggleNodeState: function(e) {
+        // expand or collapse node (chapter or sequential)
+        e.preventDefault();
+        var target = e.target || e.srcElement;
+        var row = $(target).closest('tr');
+        var children = this.getChildrenInView(row);
+
+        if (row.is('.expanded')) {
+          $(target).attr('aria-expanded', 'false');
+          $(target).find(".fa-caret-down").removeClass('fa-caret-down').addClass('fa-caret-right');
+          row.removeClass('expanded').addClass('collapsed');
+          children.hide();
+        } else {
+          $(target).attr('aria-expanded', 'true');
```

Comment:
Running an accessibility audit in Chrome (developer tools), I see that the icon tag should not use "aria-expanded".

![image](https://cloud.githubusercontent.com/assets/484484/14289599/77fea398-fb29-11e5-8d4f-7b37dc4d3e2d.png)

https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_10

You should write a bok choy accessibility test for this schedule tree. All you need to is add all the units to the try and call the "verify accessibility" method-- it's very easy if you have existing bok choy tests and page objects. Hopefully it would have caught this issue. There are also ways to write Jasmine unit tests for verifying accessibility.

See https://openedx.atlassian.net/wiki/display/TE/Automated+Accessibility+Tests and contact @clytwynec if you have any questions.